cdk/lib/__snapshots__/gatehouse.test.ts.snap

```diff
@@ -348,11 +349,7 @@ exports[`The Gatehouse stack matches the snapshot 1`] = `
                 "Fn::Join": [
                   "",
                   [
-                    "arn:aws:kinesis:",
-                    {
-                      "Ref": "AWS::Region",
-                    },
-                    ":",
+                    "arn:aws:kinesis:eu-west-1:",
```

Comment:
just curious, what was wrong with the way it was ?

Reply:
It's triggered by my adding `env: {region: 'eu-west-1'}` to `cdk/lib/gatehouse.test.ts` to make it more similar to the production config.  I don't know why that would cause the yaml test doc to structure itself differently but I'm just running `yarn test -u` to build the test snapshot.

This is nothing to do with the purpose of the PR so I'll take out that change to the test config to keep it focused.